### PR TITLE
update-regex

### PR DESCRIPTION
**Pull request by ChenxiJiang333**
fix issues
https://github.com/Azure/azure-sdk-for-python/issues/25143
and
https://github.com/Azure/azure-sdk-for-python/issues/31916

### CODE REVIEW

File: docs-ref-services/latest/commerce.md [hunk 1]
```diff
@@ -63,4 +63,4 @@ usage_iterator = commerce_client.usage_aggregates.list(
 ```
 
 > [!div class="nextstepaction"]
-> [Explore the Management APIs](/python/api/overview/azure/commerce/management)
\ No newline at end of file
+> [Explore the Management APIs](/python/api/azure-containerregistry)
```
Comment (lmazuel) [inline docs-ref-services/latest/commerce.md:66]: Doesn't feel right

File: docs-ref-services/preview/commerce.md [hunk 2]
```diff
@@ -66,4 +66,4 @@ usage_iterator = commerce_client.usage_aggregates.list(
 ```
 
 > [!div class="nextstepaction"]
-> [Explore the Management APIs](/python/api/overview/azure/commerce/management)
\ No newline at end of file
+> [Explore the Management APIs](/python/api/azure-containerregistry)
```
Comment (lmazuel) [inline docs-ref-services/preview/commerce.md:69]: Doesn't feel right

File: docs-ref-services/latest/scheduler.md [hunk 3]
```diff
@@ -71,4 +71,4 @@ scheduler_client.job_collections.create_or_update(
 ```
 
 > [!div class="nextstepaction"]
-> [Explore the Management APIs](/python/api/overview/azure/scheduler/management)
\ No newline at end of file
+> [Explore the Management APIs](/python/api/overview/azure/scheduler/management/resourcemanagement-scheduler)
```
Comment (msyyc) [inline docs-ref-services/latest/scheduler.md:74]: seems not right

File: docs-ref-services/preview/cost-management-+-billing.md [hunk 4]
```diff
@@ -20,5 +20,5 @@ ms.technology: azure
 pip install azure-mgmt-billing
 ```
 > [!div class="nextstepaction"]
-> [Explore the Management APIs](/python/api/overview/azure/billing/management)
-
+> [Explore the Management APIs](/python/api/azure-mgmt-billing/)
```
Comment (msyyc) [inline docs-ref-services/preview/cost-management-+-billing.md:23]:
```suggestion
> [Explore the Management APIs](/python/api/azure-mgmt-billing)
```

**Comment by ChenxiJiang333** [comment]
@lmazuel @msyyc I have updated by your comments, please re-review. Thanks a lot.

**Comment by msyyc** [comment]
@danieljurek Could you help merge the fix PR? Thanks!

**Comment by danieljurek** [comment]
Serice level overview pages are updated nightly and so these files will have merge conflicts. The automation which updates these runs at ~4pm Redmond time. Fix these merge conflicts and I can have a look again and merge the next morning.

**Comment by ChenxiJiang333** [comment]
updated to https://github.com/MicrosoftDocs/azure-docs-sdk-python/pull/1818

**Pull request closed without merging**